op-program/client/interop/interop_test.go

```diff
@@ -908,10 +912,11 @@ func (t *stubTasks) ExpectBuildDepositOnlyBlock(
 }
 
 type staticConfigSource struct {
-	rollupCfgs   []*rollup.Config
-	chainConfigs []*params.ChainConfig
-	depset       *depset.StaticConfigDependencySet
-	chainIDs     []eth.ChainID
+	rollupCfgs     []*rollup.Config
+	chainConfigs   []*params.ChainConfig
+	l1ChainConfigs []*params.ChainConfig
```

Comment:
(nit) I think this can just be a single value

```suggestion
	l1ChainConfig *params.ChainConfig
```

Reply:
Yep, interop chains must share the same L1. This applies everywhere where currently a slice of L1 configs is used.

Reply:
6fb102ae64cc4d8873baa30b0e116fe7a41e64dd